.github/workflows/build_for_riscv.yml

```diff
@@ -67,6 +67,10 @@ jobs:
           cmake -DCMAKE_BUILD_TYPE=Release .. && make -j && make install
           wasmedge -v
           wasmedgec -v
+          wasmedge compile -h
```

Comment:
The CI Failed. Does the `wasmedge compile -h` return non-zero?

Reply:
Yes you are right, `wasmedge compile -h` return non-zero. Let me fix this out.

Reply:
@hydai It should be fixed now